server/data/mappers/prisonerSearchSummaryMapper.ts

```diff
@@ -0,0 +1,15 @@
+import type { PrisonerSearchSummary } from 'viewModels'
```

Comment:
I introduced the `PrisonerSearchSummary` view model type the other day. This is the first PR that actually starts using it, hence this mapper.
A `PrisonerSearchSummary` is basically just a `PrisonerSummary` (built from a `Prisoner`) with 2 additional fields - `hasCiagInduction` and `hasActionPlan`, where these 2 new fields will be used on the Prisoner List screen to determine the status (Needs Plan)